mobile/cmp/panel/Panel.js

```diff
@@ -10,6 +10,7 @@ import {castArray, omitBy} from 'lodash';
 import {elemFactory, HoistComponent, LayoutSupport} from '@xh/hoist/core';
 import {vbox, vframe} from '@xh/hoist/cmp/layout';
 import {toolbar} from '@xh/hoist/mobile/cmp/toolbar';
+import {loadingIndicator} from '@xh/hoist/desktop/cmp/loadingindicator';
 import {mask} from '@xh/hoist/desktop/cmp/mask';
```

Comment:
YIKES -- we are already importing the desktop mask in to mobile?!?  Need to fix this ASAP

Reply:
OH WOW. I'll fix and update in a separate PR